front/lib/api/chat.ts

```diff
@@ -104,116 +106,135 @@ export async function getChatSession(
   };
 }
 
+export async function getChatMessage(
+  sId: string,
```

Comment:
let's call it `messageId` we generally have `sId` on the objects but when referred from the outside and in general in function signatures we use something more explciit :+1: